README.md

```diff
@@ -22,7 +21,26 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### Basic Configuration
+In order to capture and report all deprecation warnings at the end of the test run, add the following to your test setup
+```ruby
+require 'active_support/deprecation_test_helper'
+ActiveSupport::DeprecationTestHelper.configure
+```
+
+### Whitelisting Deprecation Warnings
+If you have some deprecations warnings that you're not going to resolve, and would like to not be alerted to them at the end
+of your test run, you can allow them with the following
+
+```ruby
+ActiveSupport::DeprecationTestHelper.allow_warning("The full deprecation warning string")
```

Comment:
By requiring this gem as a whole it silences all deprecation warnings from being visible inline in test output.  At the very end of the test run, a report is produced that prints out all the deprecation warnings that were not expected to exist, alerting the user to the unexpected deprecation warnings.  So I think `allow_warning` is most appropriate here because you're saying that you're allowing and expecting this deprecation warning to exist.

Reply:
I still think there is room for confusion with the terminology but I don't feel that strongly about it and if no one else finds it confusing, I'm okay as is.